content/200-concepts/100-components/02-prisma-client/055-aggregation-grouping-summarizing.mdx

```diff
@@ -602,9 +602,9 @@ model Play {
   id       Int   @id @default(autoincrement())
   score    Int?  @default(0)
   playerId Int?
-  player   User? @relation(fields: playerId, references: id)
-  gameId Int?
-  game Game? @relation(fields: gameId, references: id)
+  player   User? @relation(fields: [playerId], references: [id])
+  gameId   Int?
+  game     Game? @relation(fields: [gameId], references: [id])
```

Comment:
Do the brackets matter here? Does the formatter auto correct to this format?

Reply:
@janpio I didn't use the formatter but it says [here](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#relation) that the type of `fields` and `references` should be `FieldReference[]`, so I assumed that the brackets mattered here. The brackets are also used in the rest of the docs so even if they're not required, I'm guessing that it's better to have them to make the code style across docs more coherent.

Reply:
Fair - both work, but we have generally used square brackets in the docs. I see a few places where this isn't the case but we can clean that up.